docs/developer/conventions/code/log.rst

```diff
@@ -53,27 +61,38 @@ Finer      no unless configured             common when entering, returning, or
 Finest     no unless configured             most verbose output
 ========== ================================ ====================================================
 
-A convenience method exists in Logger for each of those levels::
+A convenience method exists in Logger for each of those levels:
+
+.. code-block:: java
    
    LOGGER.info("There is a message of interest for ordinary user");
    
-
 Do not use the logging info level as a replacement of System.out.println for displaying debug information to the console.
    
-The info level is for end users. Use the fine, finer or finest levels for debug information, and setup yours $JAVA_HOME/jre/lib/logging.properties file accordingly (see Logging Configuration below).
+The INFO level is for end users. Use the FINE, FINER or FINEST levels for debug information, and setup yours :file:`logging.properties` file accordingly (see Logging Configuration below).
```

Comment:
You seem to have deleted the Logging Configuration section below that is referenced on this line. Is there still a section on configuration somewhere?